### PR TITLE
Removing legacy resolver and updating wrapanapi

### PR DESCRIPTION
**Pull request by omkarkhatavkar**
Conflicts arrived because of urllib package as selenium uses the updated urllib package and wrapanapi does not with indirect dependancy of boto3 and botocore. bumping the boto3 and botocore will resolve this issue. 

We have PR for this in wrapanapi  https://github.com/RedHatQE/wrapanapi/pull/451

### CODE REVIEW

**Comment by omkarkhatavkar** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_product.py -k test_positive_end_to_end
airgun: 668

**Comment by omkarkhatavkar** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_product.py -k test_positive_end_to_end
airgun: 668

File: requirements.txt [hunk 1]
```diff
@@ -25,9 +25,10 @@ tenacity==8.0.1
 testimony==2.2.0
 unittest2==1.1.0
 wait-for==1.2.0
-wrapanapi==3.5.11
+
 
 # Get airgun, nailgun and upgrade from master
+git+https://github.com/omkarkhatavkar/wrapanapi.git@unpin_boto#egg=wrapanapi
```
Comment (devendra104) [inline requirements.txt:31]: can you update this one?
```suggestion
git+https://github.com/RedHatQE/wrapanapi.git@unpin_boto#egg=wrapanapi
```

File: requirements.txt [hunk 2]
```diff
@@ -25,9 +25,10 @@ tenacity==8.0.1
 testimony==2.2.0
 unittest2==1.1.0
 wait-for==1.2.0
-wrapanapi==3.5.11
```
Comment (Gauravtalreja1) [inline requirements.txt:28]: placeholder comment, resolve when new wrapanapi release with fix https://github.com/RedHatQE/wrapanapi/pull/451 is replaced here

**Comment by jyejare** [comment]
Let's see if WrapanAPI maintainers accept the unpinning of boto packages! I think @mshriver can take that decision!

File: requirements.txt (hunk 1 above)
Comment (omkarkhatavkar) [reply requirements.txt:31]: As https://github.com/SatelliteQE/robottelo/pull/9370#discussion_r812747550 we will pin the latest version once the wrapanapi PR get merged

Comment (mshriver) [reply requirements.txt:31]: https://pypi.org/project/wrapanapi/   3.5.12 is available now

Comment (omkarkhatavkar) [reply requirements.txt]: Done ! bump to 3.5.12

**Pull request merged**